projects/templates/src/lib/components/po-page-login/po-page-login-base.component.ts

```diff
@@ -968,19 +968,22 @@ export abstract class PoPageLoginBaseComponent implements OnDestroy {
     }
 
     if (this.authenticationUrl) {
+      this._loading = true;
       this.loginSubscription = this.loginService
         .onLogin(this.authenticationUrl, this.authenticationType, loginForm)
         .subscribe(
           data => {
             this.setValuesToProperties();
             sessionStorage.setItem('PO_USER_LOGIN', JSON.stringify(data));
             this.openInternalLink('/');
+            this._loading = false;
           },
           error => {
             if (error.error.code === '400' || error.error.code === '401') {
               this.setValuesToProperties(error);
               this.redirectBlockedUrl(this.exceededAttemptsWarning, this.blockedUrl);
             }
+            this._loading = false;
```

Comment:
Utilizar o this.loading no lugar do this._loading forçando a passar pelo set da propriedade e isolando a propriedade privada.

Reply:
Olá @alinelariguet, ajuste realizado!